requirements_for_test.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
-black==23.10.1  # Also update `.pre-commit-config.yaml` if this changes
+black==24.3.0  # Also update `.pre-commit-config.yaml` if this changes
```

Comment:
You need to also update the .pre-commit-config.yaml file please

Reply:
Done in [74088cc](https://github.com/alphagov/notifications-antivirus/pull/158/commits/74088ccee88d2538d58f04c6e909d8b7b3f3aed1)